Patches mrxvt to use the same color cube as xterm if the XTERM_COLORS variable
is defined. Read the file xterm16.vim for details.

--- src/init.c.orig	2005-05-28 13:46:50.000000000 -0500
+++ src/init.c	2005-05-28 14:03:27.000000000 -0500
@@ -103,10 +103,230 @@
 	"cyan",			/* 6/14: bright cyan	(#00FFFF) */
 	"white",			/* 7/15: bright white   (#FFFFFF) */
 #ifdef TTY_256COLOR
+# ifdef XTERM_COLORS
+	/* use the same color cube as xterm. 16-231 is a more or less uniform
+	 * rgb ramp, and 231-255 is a greyscale ramp */
+	"rgb:00/00/00",		/* default ff6-255 color table	 */
+	"rgb:00/00/5f",		/* consists of 6 6x6 colour cubes */
+	"rgb:00/00/87",		/* and a 24 greyscale ramp w/o	*/
+	"rgb:00/00/af",		/* black or white			*/
+	"rgb:00/00/d7",
+	"rgb:00/00/ff",
+	"rgb:00/5f/00",
+	"rgb:00/5f/5f",
+	"rgb:00/5f/87",
+	"rgb:00/5f/af",
+	"rgb:00/5f/d7",
+	"rgb:00/5f/ff",
+	"rgb:00/87/00",
+	"rgb:00/87/5f",
+	"rgb:00/87/87",
+	"rgb:00/87/af",
+	"rgb:00/87/d7",
+	"rgb:00/87/ff",
+	"rgb:00/af/00",
+	"rgb:00/af/5f",
+	"rgb:00/af/87",
+	"rgb:00/af/af",
+	"rgb:00/af/d7",
+	"rgb:00/af/ff",
+	"rgb:00/d7/00",
+	"rgb:00/d7/5f",
+	"rgb:00/d7/87",
+	"rgb:00/d7/af",
+	"rgb:00/d7/d7",
+	"rgb:00/d7/ff",
+	"rgb:00/ff/00",
+	"rgb:00/ff/5f",
+	"rgb:00/ff/87",
+	"rgb:00/ff/af",
+	"rgb:00/ff/d7",
+	"rgb:00/ff/ff",
+	"rgb:5f/00/00",
+	"rgb:5f/00/5f",
+	"rgb:5f/00/87",
+	"rgb:5f/00/af",
+	"rgb:5f/00/d7",
+	"rgb:5f/00/ff",
+	"rgb:5f/5f/00",
+	"rgb:5f/5f/5f",
+	"rgb:5f/5f/87",
+	"rgb:5f/5f/af",
+	"rgb:5f/5f/d7",
+	"rgb:5f/5f/ff",
+	"rgb:5f/87/00",
+	"rgb:5f/87/5f",
+	"rgb:5f/87/87",
+	"rgb:5f/87/af",
+	"rgb:5f/87/d7",
+	"rgb:5f/87/ff",
+	"rgb:5f/af/00",
+	"rgb:5f/af/5f",
+	"rgb:5f/af/87",
+	"rgb:5f/af/af",
+	"rgb:5f/af/d7",
+	"rgb:5f/af/ff",
+	"rgb:5f/d7/00",
+	"rgb:5f/d7/5f",
+	"rgb:5f/d7/87",
+	"rgb:5f/d7/af",
+	"rgb:5f/d7/d7",
+	"rgb:5f/d7/ff",
+	"rgb:5f/ff/00",
+	"rgb:5f/ff/5f",
+	"rgb:5f/ff/87",
+	"rgb:5f/ff/af",
+	"rgb:5f/ff/d7",
+	"rgb:5f/ff/ff",
+	"rgb:87/00/00",
+	"rgb:87/00/5f",
+	"rgb:87/00/87",
+	"rgb:87/00/af",
+	"rgb:87/00/d7",
+	"rgb:87/00/ff",
+	"rgb:87/5f/00",
+	"rgb:87/5f/5f",
+	"rgb:87/5f/87",
+	"rgb:87/5f/af",
+	"rgb:87/5f/d7",
+	"rgb:87/5f/ff",
+	"rgb:87/87/00",
+	"rgb:87/87/5f",
+	"rgb:87/87/87",
+	"rgb:87/87/af",
+	"rgb:87/87/d7",
+	"rgb:87/87/ff",
+	"rgb:87/af/00",
+	"rgb:87/af/5f",
+	"rgb:87/af/87",
+	"rgb:87/af/af",
+	"rgb:87/af/d7",
+	"rgb:87/af/ff",
+	"rgb:87/d7/00",
+	"rgb:87/d7/5f",
+	"rgb:87/d7/87",
+	"rgb:87/d7/af",
+	"rgb:87/d7/d7",
+	"rgb:87/d7/ff",
+	"rgb:87/ff/00",
+	"rgb:87/ff/5f",
+	"rgb:87/ff/87",
+	"rgb:87/ff/af",
+	"rgb:87/ff/d7",
+	"rgb:87/ff/ff",
+	"rgb:af/00/00",
+	"rgb:af/00/5f",
+	"rgb:af/00/87",
+	"rgb:af/00/af",
+	"rgb:af/00/d7",
+	"rgb:af/00/ff",
+	"rgb:af/5f/00",
+	"rgb:af/5f/5f",
+	"rgb:af/5f/87",
+	"rgb:af/5f/af",
+	"rgb:af/5f/d7",
+	"rgb:af/5f/ff",
+	"rgb:af/87/00",
+	"rgb:af/87/5f",
+	"rgb:af/87/87",
+	"rgb:af/87/af",
+	"rgb:af/87/d7",
+	"rgb:af/87/ff",
+	"rgb:af/af/00",
+	"rgb:af/af/5f",
+	"rgb:af/af/87",
+	"rgb:af/af/af",
+	"rgb:af/af/d7",
+	"rgb:af/af/ff",
+	"rgb:af/d7/00",
+	"rgb:af/d7/5f",
+	"rgb:af/d7/87",
+	"rgb:af/d7/af",
+	"rgb:af/d7/d7",
+	"rgb:af/d7/ff",
+	"rgb:af/ff/00",
+	"rgb:af/ff/5f",
+	"rgb:af/ff/87",
+	"rgb:af/ff/af",
+	"rgb:af/ff/d7",
+	"rgb:af/ff/ff",
+	"rgb:d7/00/00",
+	"rgb:d7/00/5f",
+	"rgb:d7/00/87",
+	"rgb:d7/00/af",
+	"rgb:d7/00/d7",
+	"rgb:d7/00/ff",
+	"rgb:d7/5f/00",
+	"rgb:d7/5f/5f",
+	"rgb:d7/5f/87",
+	"rgb:d7/5f/af",
+	"rgb:d7/5f/d7",
+	"rgb:d7/5f/ff",
+	"rgb:d7/87/00",
+	"rgb:d7/87/5f",
+	"rgb:d7/87/87",
+	"rgb:d7/87/af",
+	"rgb:d7/87/d7",
+	"rgb:d7/87/ff",
+	"rgb:d7/af/00",
+	"rgb:d7/af/5f",
+	"rgb:d7/af/87",
+	"rgb:d7/af/af",
+	"rgb:d7/af/d7",
+	"rgb:d7/af/ff",
+	"rgb:d7/d7/00",
+	"rgb:d7/d7/5f",
+	"rgb:d7/d7/87",
+	"rgb:d7/d7/af",
+	"rgb:d7/d7/d7",
+	"rgb:d7/d7/ff",
+	"rgb:d7/ff/00",
+	"rgb:d7/ff/5f",
+	"rgb:d7/ff/87",
+	"rgb:d7/ff/af",
+	"rgb:d7/ff/d7",
+	"rgb:d7/ff/ff",
+	"rgb:ff/00/00",
+	"rgb:ff/00/5f",
+	"rgb:ff/00/87",
+	"rgb:ff/00/af",
+	"rgb:ff/00/d7",
+	"rgb:ff/00/ff",
+	"rgb:ff/5f/00",
+	"rgb:ff/5f/5f",
+	"rgb:ff/5f/87",
+	"rgb:ff/5f/af",
+	"rgb:ff/5f/d7",
+	"rgb:ff/5f/ff",
+	"rgb:ff/87/00",
+	"rgb:ff/87/5f",
+	"rgb:ff/87/87",
+	"rgb:ff/87/af",
+	"rgb:ff/87/d7",
+	"rgb:ff/87/ff",
+	"rgb:ff/af/00",
+	"rgb:ff/af/5f",
+	"rgb:ff/af/87",
+	"rgb:ff/af/af",
+	"rgb:ff/af/d7",
+	"rgb:ff/af/ff",
+	"rgb:ff/d7/00",
+	"rgb:ff/d7/5f",
+	"rgb:ff/d7/87",
+	"rgb:ff/d7/af",
+	"rgb:ff/d7/d7",
+	"rgb:ff/d7/ff",
+	"rgb:ff/ff/00",
+	"rgb:ff/ff/5f",
+	"rgb:ff/ff/87",
+	"rgb:ff/ff/af",
+	"rgb:ff/ff/d7",
+	"rgb:ff/ff/ff",
+# else
 	"rgbi:0/0/0",		/* default 16-255 color table	 */
 	"rgbi:0/0/.2",		/* consists of 6 6x6 colour cubes */
 	"rgbi:0/0/.4",		/* and a 24 greyscale ramp w/o	*/
-	"rgbi:0/0/.6",		/* black or white				*/
+	"rgbi:0/0/.6",		/* black or white			*/
 	"rgbi:0/0/.8",
 	"rgbi:0/0/1",
 	"rgbi:0/.2/0",
@@ -319,7 +539,8 @@
 	"rgbi:1/1/.6",
 	"rgbi:1/1/.8",
 	"rgbi:1/1/1",
-	"rgb:08/08/08",
+# endif
+	"rgb:08/08/08", /* xterm, rxvt, mrxvt use the same greyscale ramp */
 	"rgb:12/12/12",
 	"rgb:1c/1c/1c",
 	"rgb:26/26/26",
